src/styles/_catalog.scss

```diff
@@ -1,22 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: 200px;
+  gap: 48px;
+  justify-content: center;
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
+
+    grid-template-columns: auto;
     align-items: center;
-    width: 200px;
```

Comment:
you can put 200px in a variable and use it in other cases

Reply:
Okey without problem